macos-guest-virtualbox.sh

```diff
@@ -217,45 +232,66 @@ if [[ -n "$(cygcheck -V 2>/dev/null)" ]]; then
             cmd.exe /d /s /c call VBoxManage.exe "$@"
         }
     else
-        cmd_path_VBoxManage='C:\Program Files\Oracle\VirtualBox\VBoxManage.exe'
-        echo "Can't find VBoxManage in PATH variable,"
-        echo "checking ${cmd_path_VBoxManage}"
-        if [[ -n "$(cmd.exe /d /s /c call "${cmd_path_VBoxManage}" -v 2>/dev/null)" ]]; then
-            function VBoxManage() {
-                cmd.exe /d /s /c call "${cmd_path_VBoxManage}" "$@"
-            }
-            echo "Found VBoxManage"
-        else
-            echo "Please make sure VirtualBox version 5.2 or higher is installed, and that"
-            echo "the path to the VBoxManage.exe executable is in the PATH variable, or assign"
-            echo "in the script the full path including the name of the executable to"
-            echo -e "the variable ${highlight_color}cmd_path_VBoxManage${default_color}"
-            exit
-        fi
+    	# drives A and B are last because they are reserved for floppy drives
+	declare -a driveletters={C..Z} && driveletters+=("A" "B")
```

Comment:
`declare -a driveletters={C..Z} ={C..Z}` seems to expand `driveletters={C..Z}` to `Z` only on bash version `4.4.12(3)-release`